suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,27 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name == 'start':
+            schema_validators[DocumentNames.start].validate(doc)
+            current_col = self._run_start_collection
+            revisions_col = self._run_start_collection_revisions
+            old = current_col.find_one({'uid': doc['uid']})
+            old.pop('_id')
+            target_uid_docs = revisions_col.find({'uid': doc['uid']})
```

Comment:
```suggestion
            target_uid_docs = revisions_col.find({'uid': doc['document.uid']})
```
I believe that by doing this, we can avoid the need to keep a copy of the uid at the top level of `wrapped`.

Reply:
```suggestion
            target_uid_docs = revisions_col.find({'document.uid': doc['uid']})
```
You mean this right?